Common subdirectories: libfcgi-2.4.0.orig/cgi-fcgi and libfcgi-2.4.0/cgi-fcgi
Common subdirectories: libfcgi-2.4.0.orig/doc and libfcgi-2.4.0/doc
Common subdirectories: libfcgi-2.4.0.orig/examples and libfcgi-2.4.0/examples
Common subdirectories: libfcgi-2.4.0.orig/images and libfcgi-2.4.0/images
Common subdirectories: libfcgi-2.4.0.orig/include and libfcgi-2.4.0/include
Common subdirectories: libfcgi-2.4.0.orig/java and libfcgi-2.4.0/java
Common subdirectories: libfcgi-2.4.0.orig/libfcgi and libfcgi-2.4.0/libfcgi
diff -du libfcgi-2.4.0.orig/Makefile.am libfcgi-2.4.0/Makefile.am
--- libfcgi-2.4.0.orig/Makefile.am	2001-12-22 14:05:39.000000000 +0100
+++ libfcgi-2.4.0/Makefile.am	2014-07-19 16:56:21.935216540 +0200
@@ -4,7 +4,7 @@
 #  $Id: Makefile.am,v 1.7 2001/12/22 13:05:39 robs Exp $
 #
 
-SUBDIRS = libfcgi cgi-fcgi examples include
+SUBDIRS = libfcgi cgi-fcgi include
 
 include_HEADERS = fcgi_config.h
 
Common subdirectories: libfcgi-2.4.0.orig/perl and libfcgi-2.4.0/perl
Common subdirectories: libfcgi-2.4.0.orig/Win32 and libfcgi-2.4.0/Win32
